client/packages/cli/src/index.js

```diff
@@ -943,36 +962,44 @@ function indexingJobCompletedMessage(job) {
       const samples = formatSamples(job.invalid_triples_sample);
       const longestValue = samples.reduce(
         (acc, { value }) => Math.max(acc, value.length),
-        // Start with length of label
         label.length,
       );
 
+      const columns = [
+        { header: 'etype', width: 15, getValue: () => etype },
```

Comment:
```suggestion
        { header: 'namespace', width: 15, getValue: () => etype },
```

I think we typically call this a `namespace` (or sometimes `table`) in user-facing areas.